SourceLink.Embed.PaketFiles/SourceLink.Embed.PaketFiles.targets

```diff
@@ -1,6 +1,6 @@
 ﻿<Project>
   <PropertyGroup>
-    <BuildDependsOn Condition="$(DebugType) == 'portable' or $(DebugType) == 'embedded'">EmbedPaketFiles;$(BuildDependsOn)</BuildDependsOn>
+    <BuildDependsOn>EmbedPaketFiles;$(BuildDependsOn)</BuildDependsOn>
```

Comment:
Are Windows PDB files also going to support embedding source files too with 15.3? cc @tmat
https://github.com/ctaggart/SourceLink/blob/v2/SourceLink.Embed.AllSourceFiles/SourceLink.Embed.AllSourceFiles.targets#L3 should also be updated to match this, one way or the other.